modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -221,6 +224,65 @@ public void processOpts() {
         }
     }
 
+    @Override
+    public String toModelImport( String name){
```

Comment:
please add some docs to describe what this does, with an example

Reply:
I added the doc to the new method `toModelImportMap`